node/src/runtime/mainnet.rs

```diff
@@ -100,11 +95,53 @@ pub fn testnet_genesis(
 	// (PUSH1 0x00 PUSH1 0x00 REVERT)
 	let revert_bytecode = vec![0x60, 0x00, 0x60, 0x00, 0xFD];
 
-	let stakers: Vec<_> = if nominate {
-		endowed_accounts.iter().map(|i| i.0.clone()).collect()
-	} else {
-		initial_authorities.iter().map(|i| i.id.clone()).collect()
+	let session_keys = SessionKeys {
+		aura: sr25519::Public::from_string(
+			"0x26c157b927d4dcc5a8f02ecaa6270052a7b7f228ee401436b07dc6b3de232a29",
+		)
+		.unwrap()
+		.into(),
+		grandpa: ed25519::Public::from_string(
+			"0x439857916bd7b0b49293bb52742187295a45d11b8919d43a4c6a7ccce0cb4d34",
+		)
+		.unwrap()
+		.into(),
+		im_online: sr25519::Public::from_string(
+			"0x16909c2879b8fcacec6ceb5505219870bbaddd9dc8cafc9437c818f92e144735",
+		)
+		.unwrap()
+		.into(),
+		beefy: ecdsa::Public::from_string(
+			"0x027f5ad307acaa5cda676a6e2915c8ec74a412279b0dedd8e9fe6fe4cae3c6f766",
+		)
+		.unwrap()
+		.into(),
 	};
+	let validator_identity: ValidatorIdentity = ValidatorIdentity {
+		id: parse_account_id("qHWFTG53dT7WvVa4HeGgrAwYNPDs6WFvzgkwxtJbQzJyjQH1S"),
+		session_keys,
+	};
+
+	let multisig_owners: Vec<_> = [
+		"5H9a1Q4rqzEK1SU5gFZBFjdBUEvCSGxJb7z9pRoE3veut153", // Raymond
```

Comment:
I'd suggest not to commit names @akorchyn 

Lets just post it somewhere in our channels instead.

Reply:
The code would be reverted, it just to create json